src/status_im/data_store/realm/schemas/account/v19/core.cljs

```diff
@@ -1,29 +1,28 @@
 (ns status-im.data-store.realm.schemas.account.v19.core
   (:require [status-im.data-store.realm.schemas.account.v11.chat :as chat]
-            [status-im.data-store.realm.schemas.account.v1.chat-contact :as chat-contact] 
+            [status-im.data-store.realm.schemas.account.v1.chat-contact :as chat-contact]
             [status-im.data-store.realm.schemas.account.v19.contact :as contact]
-            [status-im.data-store.realm.schemas.account.v1.discover :as discover] 
+            [status-im.data-store.realm.schemas.account.v19.discover :as discover]
+            [status-im.data-store.realm.schemas.account.v1.kv-store :as kv-store]
```

Comment:
@yenda in the meantime (while your PR was opened), `kv-store` dependency was already removed (because it's not used anymore), please don't bring it back :)

Reply:
ok will remove